### ISSUE TITLE
CondaUpgradeError about "newer version of conda is required" is confusing: unclear how to update conda

### ISSUE THREAD

**Issue by jorisvandenbossche** [conda/conda#7788]
### Steps to Reproduce

I am using conda-canary conda 4.6, and to test something out, reverted to conda 4.5 with `conda install conda=4.5`.

However, now I get:

```
(base) joris@joris-XPS-13-9350:~/scipy$ conda install conda-canary::conda=4.6.0b0
Solving environment: failed

CondaUpgradeError: This environment has previously been operated on by a conda version that's newer
than the conda currently being used. A newer version of conda is required.
  target environment location: /home/joris/miniconda3
  current conda version: 4.5.11
  minimum conda version: 4.6
```

So even for the command to upgrade conda again, I get this error. So how is the user expected to update the conda version again to get rid of the error? 
(the way that I updated conda again was by removing the code that raises this error in the installed package)

## Expected Behavior

I see that https://github.com/conda/conda/pull/7780 added a `context.allow_conda_downgrades`, which can be used I suppose? 
But still, I think the error message should clearly indicate how to upgrade conda again to resolve the problem.

**Comment by kalefranz** [comment]
The logic here must just be *wrong* somewhere.  The idea is that you can always upgrade to a newer version of conda; you just can't downgrade past the current major.minor version (without using the `allow_conda_downgrades` escape hatch).

**Comment by kalefranz** [comment]
Ah, I might understand what happened.  Conda 4.5 doesn't understand the `allow_conda_downgrades` config parameter yet.  (We need to backport it probably.) As I mentioned in the gitter channel, the best solution for you right now is to remove the `# conda version: 4.6` lines from your `conda-meta/history` file.

**Comment by jorisvandenbossche** [comment]
Ah, yes, so I could downgrade without problems (conda install conda=4.5 did the trick), but then upgrading not anymore. 
Now I understand more the point of the error message and the config flag ! :)

**Comment by kalefranz** [comment]
Yes, it'll only be in 4.6.0b2 and the next 4.5.x.  That's why your best option right now is to just manually remove those `# conda version:` lines in `conda-meta/history`.  The path you followed to get into this state is definitely still a bug though.  We need to fix it.

**Comment by jjhelmus** [comment]
@kalefranz For my own benefit is the suggestion to solve this issue:

* add logic to prevent conda from being downgraded at the major or minor version (4.6.0 -> 4.5.11 would not be allowed by 4.5.11 -> 4.5.7 would be) unless `allow_conda_downgrades` is set.
* backport the `allow_conda_downgrade` configuration to 4.5.x.

**Comment by kalefranz** [comment]
I think it might be a little more nuanced than that. I’m out for the next two days but let’s try to chat through it on Friday.

**Comment by nehaljwani** [comment]
With conda 4.5.12 (available in conda-canary), one can add the following to `~/.condarc` to tell conda to relax:
```
allow_conda_downgrades: true
```

**Comment by msarahan** [comment]
conda 4.5.12 is in defaults.  I'm closing this issue.  If anyone has further concerns, we can reopen or open a new issue.

**Comment by Carlo-Carandang** [comment]
> Ah, I might understand what happened. Conda 4.5 doesn't understand the `allow_conda_downgrades` config parameter yet. (We need to backport it probably.) As I mentioned in the gitter channel, the best solution for you right now is to remove the `# conda version: 4.6` lines from your `conda-meta/history` file.

Thank you- this worked for me, after having difficulties when downgrading from conda 4.6.2 with `conda install conda=4.5.11`. Removed those `conda version 4.6.2` lines from 'conda-meta/history'. When I tried to do: `conda install tensorflow-gpu`, I got the following error:

CondaUpgradeError: This environment has previously been operated on by a conda version that's newer
than the conda currently being used. A newer version of conda is required.
  target environment location: C:\Users\carandangc\Anaconda3\envs\tf-gpu
  current conda version: 4.5.11
  minimum conda version: 4.6

Update conda and try again.
    $ conda install -p "C:\Users\carandangc\Anaconda3" "conda>=4.6"

So I updated conda with the recommended `conda install -p "C:\Users\carandangc\Anaconda3" "conda>=4.6"`. After this, I was able to `conda install tensorflow-gpu` without problems, whereas I had conda installation errors with conda 4.6.2 #8046 . It appears conda 4.6 does not invoke the CondaHTTPError.
